wd_demo.au3

```diff
@@ -1209,8 +1209,12 @@ EndFunc   ;==>_Demo_NavigateCheckBanner
 
 Func SetupGecko($bHeadless)
 	_WD_Option('Driver', 'geckodriver.exe')
-	_WD_Option('DriverParams', '--log trace')
-	_WD_Option('Port', 4444)
+	Local $iPort = _WD_GetFreePort(4444)
+	If @error Then Return SetError(@error, @extended, 0)
+
+	_WD_Option('Port', $iPort)
+	_WD_Option('DriverParams', '--port=' & $_WD_PORT)
```

Comment:
Use $iPort instead of $_WD_PORT

Reply:
take a look: [$iPort uasge](https://github.com/Danp2/au3WebDriver/pull/519/commits/44384b84d308fac3a31e53fb0077b05bc9c6cb8b)